server/core/src/main/java/org/infinispan/server/core/AbstractCacheIgnoreAware.java

```diff
@@ -40,7 +42,8 @@ public void initialize(EmbeddedCacheManager cacheManager) {
 
    public boolean isCacheIgnored(String cacheName) {
       try {
-         return stateCache.containsEntry(IGNORED_CACHES_KEY, cacheName).get();
+         Optional<CacheEntry<String, Collection<String>>> values = stateCache.getEntry(IGNORED_CACHES_KEY).get();
```

Comment:
I wonder if we should be throwing an error or something if this blocks. Cause we don't want this method to ever block.

Reply:
This is just some experiment...This will need to be changed, either as a local collection backed by the cache or the ```isCacheIgnored``` returning a CompletionStage